src/test/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepTest.java

```diff
@@ -400,6 +402,41 @@ private void startJnlpProc() throws Exception {
         });
     }
 
+    @Test public void detailsExported() throws Exception {
+        story.addStep(new Statement() {
+            @Override
+            public void evaluate() throws Throwable {
+                DumbSlave s = story.j.createOnlineSlave();
+
+                WorkflowJob p = story.j.jenkins.createProject(WorkflowJob.class, "demo");
+                p.setDefinition(new CpsFlowDefinition(
+                        "node('" + s.getNodeName() + "') {\n" +
+                                "    sleep 10\n" +
```

Comment:
Wastes time in tests. Rather use the `semaphore` step and when you want to make sure we are there, use `SemaphoreStep.waitForStart`.